commit d834097ce62c7d49f928707e2b9a2d7f33c2dcf9
Author: Andy Whitcroft <apw@canonical.com>
Date:   Mon Mar 31 16:53:36 2014 +0100

    UBUNTU: SAUCE: kvm: BIOS disabled kvm support should be a warning
    
    BugLink: http://bugs.launchpad.net/bugs/1300247
    Signed-off-by: Andy Whitcroft <apw@canonical.com>

diff --git a/arch/x86/kvm/x86.c b/arch/x86/kvm/x86.c
index 699f872..46fa604 100644
--- a/arch/x86/kvm/x86.c
+++ b/arch/x86/kvm/x86.c
@@ -5853,7 +5853,7 @@ int kvm_arch_init(void *opaque)
 		goto out;
 	}
 	if (ops->disabled_by_bios()) {
-		printk(KERN_ERR "kvm: disabled by bios\n");
+		printk(KERN_WARNING "kvm: disabled by bios\n");
 		r = -EOPNOTSUPP;
 		goto out;
 	}
